docs/headless-js-android.md

```diff
@@ -98,6 +161,22 @@ return new HeadlessJsTaskConfig(
 );
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+val retryPolicy: HeadlessJsTaskRetryPolicy =
+    LinearCountingRetryPolicy(
+        3, // Max number of retry attempts
+        1000 // Delay between each retry attempt
+    )
+
+return HeadlessJsTaskConfig('SomeTaskName', Arguments.fromBundle(extras), 5000, false, retryPolicy)
```

Comment:
Single quotes are not allowed in Kotlin

Reply:
Wow.. what a mistake, sorry and thanks for the review!